src/components/Image.ts

```diff
@@ -45,31 +54,143 @@ export class Image extends Component<ImageProps, ImageChild> {
 
 	public static readonly mixed: boolean = false;
 
-	#relationshipId: string | null = null;
+	#embedMeta: {
+		location: string;
+		mime: Promise<FileMime> | null;
+		relationshipId: string | null;
+
+		extensions: {
+			svg?: {
+				location: string;
+				relationshipId: string | null;
+			};
+		};
+	};
+	get embedMeta() {
```

Comment:
Maybe I'm nitpicking here, but this (getter) property could be called simply `meta`? `Image.meta.location` seems descriptive enough, no need to specify again that this is meta about the image (or "embed", which is the same thing imho)